spec/requests/sidebar_sections_controller_spec.rb

```diff
@@ -25,6 +25,18 @@
         ["Community", "public section", "private section"],
       )
     end
+
+    it "returns Community section first even when public sections are alphabetically before it" do
+      sign_in(user)
+      Fabricate(:sidebar_section, title: "Apple", public: true)
+
+      get "/sidebar_sections.json"
+      expect(response.status).to eq(200)
+
+      titles = response.parsed_body["sidebar_sections"].map { |section| section["title"] }
+      expect(titles.first).to eq("Community")
+      expect(titles).to include("Apple")
```

Comment:
Might be better to "just" do an "`eq`" and list all the values in the array.

Something like


```suggestion
      expect(titles).to eq(["Community", "Apple", ...])
```